compiler/optimizer/OMRValuePropagation.hpp

```diff
@@ -445,6 +444,7 @@ class ValuePropagation : public TR::Optimization
    void removeRestOfBlock();
    void mustTakeException();
    void processTrees(TR::TreeTop *startTree, TR::TreeTop *endTree);
+   virtual void transformRecognizedMethod(TR::Node *node) {}
```

Comment:
I can't find where this new method is called? OMR VP needs to invoke this method to get the different behaviors we want for different projects.